### PR TITLE
Add more explicit large range example

### PR DESCRIPTION
**Pull request by Christewart**
I made the large range example more explicit, but please do proof read to make sure i'm not making any conceptual or mechanical errors.

### CODE REVIEW

File: Oracle.md [hunk 1]
```diff
@@ -95,17 +96,307 @@ unit: dollars
...
 ```
 
-Alice and Bob create a DLC with "relevant" outcomes between $10000 and $10999.
-Below $10000 Alice gets everything, and above $10999 Bob gets everything.
-They use the `R0` and `R1` values to cover the cases where BTC/USD < $10000 using `D0=0` and `D1=0`.
-They use `R0`, `R1` and `R2` to cover the cases where BTC/USD > $10999 and <= $19999, using `D1=0`, `D1=1` and `D2 in [0-9]`.
-They use `R0` and `R1` to cover the cases where BTC/USD >= $20000 and < $99999, using `D0=0` and `D1 in [2,9]`.
-Finally, they use only `D0` to cover the cases where BTC/USD >= $100000 and <= $999999, using `D0 in [1-9]`.
+Alice and Bob create a DLC with "relevant" outcomes between $10,000 and $10,999.
+These relevant outcomes will be in $500 increments between $10,000 and $10,999.
```
Comment (Tibo-lg) [inline Oracle.md]: I'm not sure I understand what you mean with the $500 increments part.

File: Oracle.md [hunk 2]
```diff
@@ -95,17 +96,307 @@ unit: dollars
...
+If the BTC/USD price is < $10,000 Bob will receive everything.
+If the BTC/USD price is >= $11,000 Alice will receive everything.
+
+
+#### Case 0: BTC/USD is < $10,000
+
+Below $10,000 Bob receives all the money in the DLC.
+Since our oracle is signing six digits, we can decompose this to `$00x,xxx`
+
+The first CET needs to be predicated on the following nonces being signed with digit `0`: `R0`,`R1`.
```
Comment (Tibo-lg) [inline Oracle.md]: > the following nonces being signed with digit

the following nonces to be used to sign the digit `0`

or something like that, it sounds like you are signing the nonces

File: Oracle.md [hunk 3]
```diff
@@ -95,17 +96,307 @@ unit: dollars
...
+
+The rest of the digits (`R2`,`R3`,`R4`,`R5`) are irrelevant for this case.
```
Comment (Tibo-lg) [inline Oracle.md]: digits -> nonces 

Or change R2... with D2... (since you kept the Ds above, maybe better to use them actually)

File: Oracle.md [hunk 4]
```diff
@@ -95,17 +96,307 @@ unit: dollars
...
+
+#### Case 1: BTC/USD is >= $11,000
+
+If the BTC/USD price is `>= $11,000` Alice receives all the money in the DLC
+Since our oracle is signing six digits, we can think of this as `$0[!1]x,xxx`
+In plain english, this means that the most significant digit is zero, second most signifcant digit is NOT 1.
+
+Since we do not have the ability to represent a logical NOT in our signing scheme, the 9 CETs need to be constructed here.
```
Comment (Tibo-lg) [inline Oracle.md]: the 9 -> 9

File: Oracle.md [hunk 5]
```diff
@@ -95,17 +96,307 @@ unit: dollars
...
+
+For Alice to receive the money, she needs `R0=0` AND (`R1=0` OR `R1=2,3,4...9`).
```
Comment (Tibo-lg) [inline Oracle.md]: Again you should be using Dx instead of Rx (I had made the confusion myself before, @nkohen suggested using the Ds which I found nice)

File: Oracle.md [hunk 6]
```diff
@@ -95,17 +96,307 @@ unit: dollars
...
+
+This means we have to construct `1 * 9 = 9` CETs for the outcome of Alice winning all the money in the DLC where BTC/USD is >= $11,000.
+
+#### Case 2: BTC/USD is >= $10,000 and <= $10,499
+
+If the BTC/USD price is `[$10,001-$10,499]` Bob receives all the money in the DLC.
+Since our oracle is signing six digits, we can think of this as `$01[0...4],xxx`
+
+In plain english, this means that the most significant digit (`R0`) is required to be `0`.
```
Comment (Tibo-lg) [inline Oracle.md]: R0 -> D0

Comment (Tibo-lg) [reply Oracle.md]: And after

File: Oracle.md [hunk 7]
```diff
@@ -95,17 +96,307 @@ unit: dollars
...
+The 2nd most significant digit (`R1`) is required to be `1`.
+
+The third most significant digit (`R2`) MUST be in the range `[5-9]`
+
+The rest of the digits (`R3`,`R4`,`R5`) can be any value.
+
+Since we do not have the ability to represent ranges, we need to construct individual CETs for each outcome.
+This means we have to construct `1 * 1 * 5 * 10 * 10 = 500` CETs for the outcome Alice's winning outcome where BTC/USD is >= $10,500 and <= $10,999
+
+### Serialization and signing of outcome values
```
Comment (Tibo-lg) [inline Oracle.md]: This part (and a big chunk after it seems) is now duplicated, is that intentional?

Comment (Tibo-lg) [reply Oracle.md]: Yeah there is definitely something weird here, you also have the introduction under there.

File: Oracle.md (hunk 1 above)
Comment (Tibo-lg) [reply Oracle.md]: So after reading until the end, I think what you are describing is a binary option? So in that case I don't see why the relevant outcomes are between $10000 and $10999. And I still don't get the $500 increments part.

**Review by Tibo-lg — CHANGES_REQUESTED** [review]
There is something weird after the description you updated, not sure what happened.

Also as @nkohen pointed out to me be careful to use `Dx` for the digit values and not the nonces.

File: Oracle.md (hunk 1 above)
Comment (Christewart) [reply Oracle.md]: The $500 increments is a random threshold I picked out of a hat. Would you prefer I do $250 increments? $100 increments? I choose $500 for the sake of brevity because it gets to be a lot to write out, but if the example is pointless if it isn't helpfull to others. Perhaps I need to reword the introduction to something like 

>Alice and Bob create a DLC with oscillating outcomes between $10,000 and $10,999.
The oscillation will occur in $500 increments between $10,000 and $10,999.
For this example, this means 1 oscillation will occur at price point $10,500. 
If the BTC/USD price is < $10,000 Bob will receive everything.
If the BTC/USD price is >= $11,000 Alice will receive everything.
